test/unit/views/MessagesSpec.scala

```diff
@@ -39,9 +39,7 @@ class MessagesSpec extends PlaySpec with Injector {
     "cds.subscription-details.tab.title.data"
   )
 
-  // TODO REMOVE keyStartsWith("ecc.reg06.") Once we receive the translations
-  def ignoreKey(key: String): Boolean =
-    key.startsWith("cds.country.") || sameTranslation.contains(key) || key.startsWith("ecc.reg06.")
```

Comment:
This change is because we're checking if Welsh translation is not the same like English. We added last time exception for messages that starts with `ecc.reg06.` to be able to keep English content for those messages.

Now we're removing this workaround